### PR TITLE
Add person-detection-0203 model

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**

### CODE REVIEW

**Comment by GalyaZalesskaya** [comment]
run tests

**Comment by GalyaZalesskaya** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

File: pytorch_toolkit/object_detection/tests/export_tests_person_detection.py [hunk 1]
```diff
@@ -48,3 +48,15 @@ class PersonDetection0202TestCase(
             True)
 ):
     """ Test case for person-detection-0202 model export. """
+
+
+class PersonDetection0203TestCase(
+        create_export_test_case(
+            'person-detection',
+            'person-detection-0203',
+            os.path.dirname(__file__) + '/../../../data/airport/annotation_example_train.json',
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/tests/export_tests_person_detection.py:57]: try to use proper annotation here and above 'annotation_person_train.json', I am fixing it in my PR. Looks like CI job is hanging on loading http://213.221.44.203/job/ote/job/precommit/job/pt_object_detection/505/consoleFull. I kill it.

File: pytorch_toolkit/object_detection/tests/expected_outputs/person-detection/person-detection-0203.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"map": 0.704}
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/tests/expected_outputs/person-detection/person-detection-0203.json:1]: for now, metric is measure in percent, 70.4 should be here

Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/tests/expected_outputs/person-detection/person-detection-0203.json:1]: for now, metric is measure in percent, 70.4 should be here

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

File: pytorch_toolkit/object_detection/model_templates/person-detection/readme.md [hunk 3]
```diff
@@ -110,9 +111,9 @@ python eval.py \
    --save-output-to ${WORK_DIR}/output_images
 ```
 
-### 7. Export PyTorch\* model to the OpenVINO™ format
+### 7. Export PyTorch\* model to the OpenVINO format
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/person-detection/readme.md:114]: ?

File: pytorch_toolkit/object_detection/model_templates/person-detection/readme.md [hunk 4]
```diff
@@ -110,9 +111,9 @@ python eval.py \
...
 
-To convert PyTorch\* model to the OpenVINO™ IR format run the `export.py` script:
+To convert PyTorch\* model to the OpenVINO IR format run the `export.py` script:
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/person-detection/readme.md:116]: ?

File: pytorch_toolkit/object_detection/model_templates/person-detection/readme.md [hunk 5]
```diff
@@ -123,7 +124,7 @@ python export.py \
 This produces model `model.xml` and weights `model.bin` in single-precision floating-point format
 (FP32). The obtained model expects **normalized image** in planar BGR format.
 
-For SSD networks an alternative OpenVINO™ representation is saved automatically to `${WORK_DIR}/export/alt_ssd_export` folder.
+For SSD networks an alternative OpenVINO representation is saved automatically to `${WORK_DIR}/export/alt_ssd_export` folder.
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/person-detection/readme.md:127]: ?

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by GalyaZalesskaya** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by GalyaZalesskaya** [comment]
run tests

**Comment by GalyaZalesskaya** [comment]
@Ilya-Krylov tests finally passed and changes from your [PR](https://github.com/openvinotoolkit/training_extensions/pull/411) with template fixes were applied. Could you please approve these changes?

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**